.gitignore

```diff
@@ -156,6 +156,17 @@ ml-20m/
 *.model
 *.mml
 nohup.out
+
+#####  kdd 2020 tutorial data folder
+scenarios/KDD2020-tutorial/data_folder.zip
+scenarios/KDD2020-tutorial/data_folder/
+scenarios/KDD2020-tutorial/.ipynb_checkpoints/
+scenarios/academic/KDD2020-tutorial/data_folder/
+scenarios/academic/KDD2020-tutorial/data_folder.zip
+scenarios/academic/KDD2020-tutorial/.ipynb_checkpoints/
+
 *.vec
 *.tsv
 *.sh
+
+tests/resources/
```

Comment:
@yueguoguo  @miguelgfierro   can you confirm that is fine to add "tests/resources/ " to the gitignore?

Reply:
Yea this is fine to me.

Reply:
I think this is ok